lib/medium-admin.php

```diff
@@ -736,7 +724,7 @@ private static function _get_migration_data() {
   /**
    * Creates a post on Medium.
    */
-  public static function cross_post($post, $medium_post, $medium_user) {
+  public static function cross_post($post, &$medium_post, $medium_user) {
```

Comment:
Ah yeah, this was from when we moved the claims call outside `cross_post`—will remove